easybuild/easyconfigs/v/VirSorter/VirSorter-20160601-foss-2016a-Perl-5.22.1.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'Tarball'
+
+name = 'VirSorter'
+version = '20160601'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'https://github.com/simroux/VirSorter'
+description = """VirSorter: mining viral signal from microbial genomic data."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/simroux/%(name)s/archive/']
+sources = ['master.zip']
+
+dependencies = [
+    ('Perl', '5.22.1'),
+    ('BioPerl', '1.6.924', '-Perl-%(perlver)s'),
+    ('HMMER', '3.1b2'),
+    ('MCL', '14.137'),
+    ('MetaGeneAnnotator', '20080819', '', True),
+    ('MUSCLE', '3.8.31'),
+    ('BLAST', '2.2.26', '-Linux_x86_64', True),
+]
+
+sanity_check_paths = {
+    'files': ['wrapper_phage_contigs_sorter_iPlant.pl'],
+    'dirs': ['Scripts'],
+}
+
+modextrapaths = {'PATH': ['', 'Scripts']}
+modaliases = {'virsorter': 'wrapper_phage_contigs_sorter_iPlant.pl'}
```

Comment:
@bedroge please use `%(namelower)s` here to make the tests happy (the test that compares the original easyconfig with the dumped easyconfig fails because of the hardcoded `virsorter` here...)